src/system_sensors.py

```diff
@@ -262,6 +277,30 @@ def remove_old_topics():
         qos=1,
         retain=False,
     )
+    mqttClient.publish(
+        topic=f"homeassistant/sensor/{deviceNameDisplay}/{deviceNameDisplay}Hostname/config",
+        payload='',
+        qos=1,
+        retain=False,
+    )
+    mqttClient.publish(
+        topic=f"homeassistant/sensor/{deviceNameDisplay}/{deviceNameDisplay}Host_Ip/config",
+        payload='',
+        qos=1,
+        retain=False,
+    )
+    mqttClient.publish(
+        topic=f"homeassistant/sensor/{deviceNameDisplay}/{deviceNameDisplay}Host_OS/config",
+        payload='',
+        qos=1,
+        retain=False,
+    )
+    mqttClient.publish(
+        topic=f"homeassistant/sensor/{deviceNameDisplay}/{deviceNameDisplay}Host_Arch/config",
+        payload='',
+        qos=1,
+        retain=False,
+    )
```

Comment:
This isn't necessary it's only because I changed the path of the existing sensors. These are new sensors so they didn't exist before I changed the paths